packages/keystone/preprocessors/customAccess.js

```diff
@@ -0,0 +1,145 @@
+const get = require('lodash/get')
+const { parseListAccess, parseFieldAccess, parseCustomAccess } = require('@keystonejs/access-control')
+const { GQL_LIST_SCHEMA_TYPE } = require('@open-condo/keystone/schema')
+
+const DEFAULT_PERMISSIONS = { read: false, update: false, create: false, delete: false }
+const IGNORED_PLUGIN_FIELDS = ['v', 'id', 'dv', 'sender', 'createdAt', 'updatedAt', 'createdBy', 'updatedBy']
```

Comment:
We also have `newId` field. Should it be here?

Reply:
That's a bad idea to ignore some fields. But I can't find any solution for keystone admin panel to make it properly work with dynamic access style